### DISCUSSION TITLE
What is your opinion on ice WM

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#424] [Q&A] [answered]
Did you consider icewm before switching to kwin and what are your thoughts on it？

**Comment by probonopd** [comment, marked as answer]
It's not Qt based. Qt is the native toolkit for helloSystem. KWin uses Qt. So helloSystem uses KWin.

**Comment by louies0623** [comment]
![image](https://user-images.githubusercontent.com/44593430/209557569-7c7d19b0-6efe-4243-8181-ef21bc08e771.png)
Recently found this FLTK, and discuss it here, what do you think.

**Reply by probonopd** [reply]
We are using KWin because it is Qt based like the rest of our user interface.

**Comment by louies0623** [comment]
So what do you think of Compiz based on Python and C++?
